chromadb/telemetry/opentelemetry/__init__.py

```diff
@@ -99,6 +99,9 @@ def otel_init(
     granularity = otel_granularity
 
 
+T = TypeVar("T", bound=Callable)
```

Comment:
Is this change just to get the types on as opposed to Any?

Reply:
yep, otherwise the types weren't working out for async functions iirc